app/models/instrument.rb

```diff
@@ -44,6 +44,10 @@ def self.reservation_only
   # Instance methods
   # -------
 
+  def time_data_for(order_detail)
```

Comment:
I don't really envision using it to display canceled. We're not planning to add the `canceled_at`, etc fields to Occupancy, so those fields will be reservation specific until we get the refactor of moving those fields onto OrderDetail.

Part of the idea is that this really returns more of a duck-type of the things that are shared between reservations and occupancy, things like `actual_start_at` (we will probably alias `entry_at` to that field to maintain the interface and are-there-problems-like-missing-times questions.

We'll keep reservation-specific things to `order_detail.reservation`.